src/newLaunch/lbp/stages/stage7.html

```diff
@@ -15,25 +15,23 @@
           Prime Launch interface. The approval takes place in the <a href.bind="multiSigWalletUri" target="_blank"
             rel="noopener noreferrer">Prime Launch multisig wallet</a>. Reviews generally take 5-7 days.</p>
 
-        <p>After being approved, the <etherscanlink hide-clipboard-button
-            address.to-view="launchConfig.launchDetails.adminAddress" text="Liquid Launch Admin"></etherscanlink>
-
+        <p>
```

Comment:
Heya, can you comment on the files (both of them), what you did to fix the issue?
The diff does not reveal that to me :(